src/main/scala/com.snowplowanalytics.snowplow/scalatracker/Payload.scala

```diff
@@ -42,6 +42,19 @@ class Payload {
   }
 
   /**
+   * Overloaded add function for Option. Don't modify payload for None
+   *
+   * @param name parameter name
+   * @param value optional parameter value
+   */
+  def add(name: String, value: Option[String]): Unit = {
```

Comment:
^^ `: Unit = ` is unnecessary (and not consistent with procedure defns elsewhere, e.g. add() above).